src/System.Configuration.ConfigurationManager/tests/System/Configuration/TimeSpanValidatorAttributeTests.cs

```diff
@@ -111,6 +111,7 @@ public void MinValue_Get()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot,"Exception messages are different")]
```

Comment:
Generally we don't look at exception messages so it's better to just fix the test that way